src/test/java/teammates/test/cases/action/StudentFeedbackResultsPageActionTest.java

```diff
@@ -22,6 +25,28 @@
  */
 public class StudentFeedbackResultsPageActionTest extends BaseActionTest {
 
+    @BeforeClass
+    public void classSetup() throws Exception {
+        addUnregStudentToCourse1();
+    }
+
+    @AfterClass
+    public void classTearDown() {
+        StudentsLogic.inst().deleteStudentCascade("idOfTypicalCourse1", "student6InCourse1@gmail.tmt");
+    }
+
+    private static void addUnregStudentToCourse1() throws Exception {
```

Comment:
Can move method to BaseActionTest since it is reused a few times in several tests.